### PR TITLE
build(release-trigger): build from official node image

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: packages/release-trigger/Dockerfile [hunk 1]
```diff
@@ -14,9 +14,33 @@
 
 # Use a multi-stage docker build to limit production dependencies.
 
+FROM node:18.20.3-bullseye AS BASE
```
Comment (meltsufin) [inline packages/release-trigger/Dockerfile:17]: Can we setup Renovate to suggest updates to this version?

File: packages/release-trigger/Dockerfile [hunk 2]
```diff
@@ -14,9 +14,33 @@
...
+RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv && \
+    echo 'export PATH="/home/node/.pyenv/bin:$PATH"' >> ~/.profile && \
+    echo 'export PATH="/home/node/.pyenv/shims:$PATH"' >> ~/.profile && \
+    echo 'eval "$(pyenv init -)"' >> ~/.profile && \
+    echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
+
+ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
+
+# Install python
+RUN pyenv install 3.10.13 && \
```
Comment (meltsufin) [inline packages/release-trigger/Dockerfile:37]: Also here, maybe Renovate can be configured to manage this version.

**Review by meltsufin — APPROVED** [review]

File: packages/release-trigger/Dockerfile (hunk 1 above)
Comment (chingor13) [reply packages/release-trigger/Dockerfile:17]: Can we do it separately? Also updating a bunch of images in #5365

File: packages/release-trigger/Dockerfile (hunk 2 above)
Comment (meltsufin) [reply packages/release-trigger/Dockerfile:37]: Yes, let's track it as a separate issue.

Comment (chingor13) [reply packages/release-trigger/Dockerfile:37]: #5366 5366

**Review by sofisl — APPROVED** [review]

**Pull request merged**